components/Layout/Page.tsx

```diff
@@ -101,19 +120,29 @@ const lightTheme = {
     tertiary: '#240090',
     quarternary: '#3500d3',
   },
+  mediaQueries: {
+    small: `@media screen and (max-width: 768px)`,
+    medium: `@media screen and (max-width: 1024px)`,
+    large: `@media screen and (max-width: 1200px)`,
+  },
 }
 
 const darkTheme = {
   bg: {
     primary: '#0C0032',
     secondary: '#190061',
-    tertiary: '#240090',
-    quarternary: '#3500d3',
+    tertiary: '#3500d3',
+    quarternary: '#240090',
```

Comment:
At one point this naming is hard to scale. It's fine for now, but you might also consider giving it descriptive names.
Bootstrap has a good example on that
ref: https://getbootstrap.com/docs/5.1/components/buttons/